### ISSUE TITLE
Terraform just broke on localstack

### ISSUE THREAD

**Issue by rayjlinden** [localstack/localstack#3270]
Actually, it is only broke for deploying lambdas.  You can also work around it by using the previous providor:

```
terraform {
  required_providers {
    aws = "<= 3.16.0"
  }
}
```

The reason it broke is it using the new aws feature for code signing lambdas.  Those new APIs are not implemented or mocked in a reasonable way in localstack.

Is there a plan to mock or implement those APIs?

┆Issue is synchronized with this [Jira Bug](https://localstack.atlassian.net/browse/LOC-347) by [Unito](https://www.unito.io/learn-more)

**Comment by whummer** [comment]
Thanks for reporting @rayjlinden . This should be fixed in #3280 - can you please pull the `latest` Docker image and give it another try? Please report here if the problem persists. Thanks

**Comment by rayjlinden** [comment]
I have been using it and it works great.  At least until a few hours ago when terraform did another release that broke things again!!!  I'll file separate report for that...
